openedx/features/ucsd_support/views.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import json
+import logging
+
+from django.core.mail import send_mail
+from django.conf import settings
+from django.views.decorators.http import require_http_methods
+from django.http import HttpResponse
+from rest_framework import status
+
+log = logging.getLogger(__name__)
+
+
+@require_http_methods(["POST"])
+def email_support(request):
+    """
+    A View that will send user support (contact-form) emails to a specific
+    account
+
+    """
+
+    data = json.loads(request.body)
+    subject = data['subject']
+    requester_name = data['requester']['name']
+    requester_email = data['requester']['email']
+
+    body = "Do Not Reply To This Email, " \
+           "It has been generated by the server on the request of a user."
+    body += "\n\n{0}".format(data['comment']['body'])
+    body += "\n\nFrom : {0}\n Email : {1}".format(requester_name,
+                                                  requester_email)
+
+    response = send_mail(subject, body, settings.DEFAULT_FROM_EMAIL,
+                         ['servicedesk@ucsd.edu'], fail_silently=False)
```

Comment:
Please read about `fail_silently` and use a proper value according to our situation.